src/main/java/org/spongepowered/common/mixin/api/mcp/entity/EntityMixin_API.java

```diff
@@ -447,7 +447,7 @@ public boolean isLoaded() {
 
     @Override
     public void remove() {
-        this.isDead = true;
+        setDead();
```

Comment:
Is this an API call or an implementation (native) method call?

Reply:
`remove` is from the API and `setDead` is from NMS. There is an injection in `setDead` to fire the `DestructEntityEvent`.

Reply:
Then rename the `setDead()` to `shadow$setDead()`. It's easier/clearer to see where the method comes from that way.

Reply:
1. It will affect many other classes extending `EntityMixin_API` and calling `setDead`.
2. `setDead` is also shadowed in `EntityMixin` and `EntityMixin_Activation`.
Should I rename everywhere or just in `EntityMixin_API` and its subclasses ?

Reply:
> 1. It will affect many other classes extending `EntityMixin_API` and calling `setDead`.

Ah, never mind then. Thought it was an overall addition.